### ISSUE TITLE
BUG: ipython version check

### ISSUE THREAD

**Issue by andythomas501** [gotcha/ipdb#72]
in the file ipdb/__main__.py

line 43;
```.py
if IPython.__version__ > '0.10.2': 
    from IPython.core.debugger import Pdb, BdbQuit_excepthook 
```
it performs lexical comparison, which is incorrect in this case, e.g. for version 0.8.4 it will return true because 8 > 1.

you could easily fix it with:
```.py
if tuple(int(x) for x in IPython.__version__.split('.')) > (0, 10, 2):
    from IPython.core.debugger import Pdb, BdbQuit_excepthook
```

**Comment by sas23** [comment]
https://github.com/gotcha/ipdb/pull/76

**Comment by gotcha** [comment]
#76 was released in 0.8.2
